cluster/addons/addon-manager/Dockerfile

```diff
@@ -14,6 +14,8 @@
 
 FROM BASEIMAGE
 
+RUN apt-get update && apt-get install -y bash
```

Comment:
use `clean-install bash` instead. (it's a helper script we installed in `debian-base` which does this and also cleans up the caches afterwards.)

Reply:
Thanks, good to know. Updated now.